src/Frontend/Core/Layout/Templates/Pagination.html.twig

```diff
@@ -16,7 +16,7 @@
             {% if pagination.first %}
               {% for pagination in pagination.first %}
```

Comment:
in the bootstrap theme it is like @GhaziTriki wrote, so I think this implementation is correct